README.md

```diff
@@ -58,6 +59,7 @@ workers implement the Remote Worker API.
 
 * [bazel-remote](https://github.com/buchgr/bazel-remote) (open source, cache only)
 * [Buildbarn](https://github.com/buildbarn) (open source)
+* [buildbox-casd](https://gitlab.com/BuildGrid/buildbox/buildbox-casd) (open source)
 * [BuildBuddy](https://www.buildbuddy.io/) (commercial & open source)
 * [Buildfarm](https://github.com/bazelbuild/bazel-buildfarm) (open source)
 * [BuildGrid](https://buildgrid.build/) (open source)
```

Comment:
Could you help elaborate how to differentiate `buildbox-casd` vs BuildGrid? Aren't they the same project?

Reply:
buildbox-casd is a caching proxy, designed to be mainly used locally on worker machines and client systems. It provides additional functionality that only makes sense for a local server such as integration with buildbox-fuse (used by buildbox-run for local execution).

The proxy function is optional, which means that it can theoretically also be deployed as server for remote clients. This can be useful for tiny deployments. However, as it only supports local filesystem storage, this is not a real alternative for larger deployments that BuildGrid and other servers support.

It supports proxying of Execute requests and should soon also support local sandboxed execution without a remote execution service.

Reply:
SGTM. Thanks!